IdokladSdk.IntegrationTests/Tests/Clients/CreditNote/CreditNoteTests.cs

```diff
@@ -280,7 +280,7 @@ private static void ComparePatchAndGetModels(CreditNotePatchModel patchModel, Cr
             Assert.That(getModel.MyAddress.BankCode, Is.EqualTo("0600"));
             Assert.That(getModel.PaymentOptionId, Is.EqualTo(patchModel.PaymentOptionId));
             Assert.That(getModel.Prices, Is.Not.Null);
-            Assert.That(getModel.Prices.TotalWithVat, Is.Zero);
+            Assert.That(getModel.Prices.TotalWithVat, Is.Not.Zero);
```

Comment:
Tohle je hodně zvláštní změna. Před tím byl očekávaný stav úplný opak? Co se změnilo?

Reply:
nepozeráš sa na všetky zmeny pozeráš sa asi na zmeny medzi niektorými commitami, pôvodne to bolo Assert.NotZero(getModel.Prices.TotalWithVat) a v tomto commite som opravoval chybu, ktorú som tam zaviedol